docker/docker-compose.yaml

```diff
@@ -37,5 +37,13 @@ services:
     ports:
       - 8002:80
 
+  ssh:
+    container_name: 'kernelci-ssh'
+    build:
+      context: ssh
+    volumes:
+      - './storage/data:/home/kernelci/data'
+      - './ssh:/home/kernelci/.ssh'
```

Comment:
This works, but we have files there which aren't really needed:
```
$ docker-compose run -u kernelci ssh /bin/bash
kernelci@222ff8a81d96:~$ ls -l /home/kernelci/.ssh
total 8
-rw-r--r-- 1 kernelci kernelci  652 Dec 23 09:53 Dockerfile
-rw-r--r-- 1 kernelci kernelci    0 Dec 23 09:53 authorized_keys.sample
-rw-r--r-- 1 kernelci kernelci 3289 Dec 23 09:53 sshd_config
```
So maybe we should have a sub-directory, for example `docker/ssh/user-data` and then mount that in `.ssh`:
```yaml
  ssh:
    container_name: 'kernelci-ssh'
    build:
      context: ssh
    volumes:
      - './storage/data:/home/kernelci/data'
      - './ssh/user-data:/home/kernelci/.ssh'
```
The `authorized_keys.sample` file would then be in that sub-directory.